app/src/main/res/values/strings.xml

```diff
@@ -65,6 +65,10 @@
     <string name="tap_on_mic">Tap on mic to speak</string>
     <string name="know_more">Know more</string>
 
+    <string name="settings_theme">Theme Settings</string>
+    <string name="theme">Select Topic</string>
+    <string name="selection_theme"></string>
```

Comment:
Remove these 3 as we are not using themes anymore. Remove from other translation as well.